RadixWallet/Clients/TransportProfileClient/TransportProfileClient+Live.swift

```diff
@@ -14,7 +14,9 @@ extension TransportProfileClient: DependencyKey {
 				do {
 					var profile = profile
 					await profileStore.claimOwnership(of: &profile)
-					try await cloudBackupClient.claimProfileOnICloud(profile)
+					if profile.appPreferences.security.isCloudProfileSyncEnabled {
+						try? await cloudBackupClient.claimProfileOnICloud(profile)
```

Comment:
makes sense! Perhaps use `do catch` and log the error?

Reply:
Sure, logging inside the function instead